packages/p2p/src/utils/responsive.ts

```diff
@@ -0,0 +1,9 @@
+import { isMobile } from '@deriv/shared';
+
+/**
+ * Function to get the label size based on the screen size
+ * @function getFieldValueSize
+ * @returns {string} label size
+ */
+export const getFieldValueSize = (mobile_size: string, desktop_size: string): string =>
```

Comment:
```suggestion
export const getFieldValueSize = (mobile_size: string, desktop_size: string) =>
```

Do we need this return type?

Reply:
don't we normally mention return type for functions in typescript?